Source/WebKit/WebProcess/WebPage/WebPage.cpp

```diff
@@ -5633,8 +5629,6 @@ void WebPage::didChooseColor(const WebCore::Color& color)
         activeColorChooser->didChooseColor(color);
 }
 
-#endif
```

Comment:
Guess: WPE and Windows are probably crashing because this code in WebPage.cpp is not guarded by the runtime preference?

We might also need a check in `WebPageProxy::showColorPicker` to ensure a malicious web process cannot crash the UI process by sending a ShowColorPicker message.